### PR TITLE
Get Bundler output out of test suite!

### LINKED ISSUE #5463: `rake spec` (`script/test`) outputs bundle commands
I think somebody did something wrong.

### PR DESCRIPTION
**Pull request by jekyllbot**
PR automatically created for @pathawks.

```text
Get Bundler output out of test suite!

Fixes #5463

/cc: @envygeeks @ashmaroli
```

### CODE REVIEW

**Comment by pathawks** [comment]
Fixes #5463

/cc: @envygeeks @ashmaroli

---
@jekyllbot You disappoint me 😞

File: lib/jekyll/commands/new.rb [hunk 1]
```diff
@@ -137,11 +137,7 @@ def bundle_install(path)
           Jekyll::External.require_with_graceful_fail "bundler"
           Jekyll.logger.info "Running bundle install in #{path.cyan}..."
           Dir.chdir(path) do
-            if ENV["CI"]
-              system("bundle", "install", "--quiet")
-            else
-              system("bundle", "install")
-            end
+            Jekyll.logger.info `bundle install`
```
Comment (ashmaroli) [inline lib/jekyll/commands/new.rb]: This effectively removes the auto-initiation of `bundle install` while only **prompting** the user to `bundle install` which IMO, is pointless.

Comment (pathawks) [inline lib/jekyll/commands/new.rb]: Nope, it runs the command and pipes its output to `logger`.

It doesn't just print `bundle install` 😂

**Comment by pathawks** [comment]
@ashmaroli You're right. Forgot about that PR :+1:

**Pull request closed without merging**